### PR TITLE
fix: theme factory handling of null sections

### PR DESCRIPTION
**Pull request by aronshamash**
- default values don't handle `null` values well so need the extra check for section.
- Fix Render 500 errors in CI with latest TC.

### CODE REVIEW

File: packages/styleguide/src/theme/theme-factory.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ const magazineFontPicker = (
 };
 
 const headlineCasePicker = (section = "default", template = "mainstandard") =>
+  section &&
```
Comment (nehasri89) [inline packages/styleguide/src/theme/theme-factory.js]: but we are specifying a default value in the params

Comment (aronshamash) [reply packages/styleguide/src/theme/theme-factory.js]: yes it is good for `undefined` but not `null` see https://medium.com/@bmb21/default-parameters-and-nulls-in-javascript-9149c8d26b0a

Comment (parkjh) [inline packages/styleguide/src/theme/theme-factory.js]: no test...?

Comment (ValGeorgiev) [reply packages/styleguide/src/theme/theme-factory.js]: Unfortunately.. no one function in theme-factory.js has unit tests... Lets merge this one and i will put in my to do list to increase the test coverage in this file. Thanks

**Pull request merged**